libshaderc_spvc/src/spvc_c_smoke_test.c

```diff
@@ -73,6 +73,9 @@ int main() {
     printf("failed to produce glsl\n");
     return -1;
   }
+  printf("success! %lu characters of glsl\n",
```

Comment:
Can this just be %zu and we drop the cast to (unsigned long)?

Reply:
No, some old windows compiler chokes on %zu.